src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Performance/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.Azure.ServiceBus;
```

Comment:
For conformance with the existing formatting used in the other Service Bus projects, should we consider moving the `using` statements inside the namespace and adding the standard copyright header?